### DISCUSSION TITLE
Install Audiobookshelf on Debian 11

### DISCUSSION THREAD

**Discussion by witzker** [advplyr/audiobookshelf#1065] [Q&A] [answered]
Hi,
I'm new with this server stuff
I installed the [Turkey Core ISO](https://www.turnkeylinux.org/core) on esxi7

Then I followed the instructions from here
https://www.audiobookshelf.org/install
`
wget https://advplyr.github.io/audiobookshelf-ppa/audiobookshelf_2.2.1_amd64.deb
sudo apt install ./audiobookshelf_2.2.1_amd64.deb

but did not work

The i tried to install the package 
audiobookshelf_2.2.1_amd64.deb in Webmin
![grafik](https://user-images.githubusercontent.com/4120266/195942784-458a7edc-0f0a-4e0c-a26c-38d6dfaa468a.png)

![grafik](https://user-images.githubusercontent.com/4120266/195942930-0d9c80ac-87b2-478b-8e09-7bd3db3cd3e7.png)

and get this:

2022-10-14 20:48:42 (7.16 MB/s) - 'ffmpeg-git-amd64-static.tar.xz.12' saved [40997632/40997632]

tar (child): xz: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
dpkg: error processing archive /root/audiobookshelf_2.2.1_amd64.deb (--install):
 new audiobookshelf package pre-installation script subprocess returned error exit status 2
Errors were encountered while processing:
 /root/audiobookshelf_2.2.1_amd64.deb

Pls Help a newbie to get this installed
THX

**Comment by apineiro97** [comment, marked as answer]
I am not an expert, but it looks like you are missing the xz program. 

https://superuser.com/questions/801159/cannot-decompress-tar-xz-file-getting-xz-cannot-exec-no-such-file-or-direct"

running should do the trick
sudo apt-get install xz-utils
